reference/shaders-msl/flatten/push-constant.flatten.vert

```diff
@@ -0,0 +1,33 @@
+#include <metal_stdlib>
+#include <simd/simd.h>
+
+using namespace metal;
+
+struct PushMe
+{
+    float4x4 MVP;
+    float2x2 Rot;
```

Comment:
How are the packing rules for Metal? GLSL has std140/std430 layouts which would change how e.g. mat2's are packed in memory.